payloads/v8/sticker.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * Types extracted from https://discord.com/developers/docs/resources/sticker
+ */
+
+import type { Snowflake } from '../../globals';
+import type { APIUser } from './user';
+
+/**
+ * https://discord.com/developers/docs/resources/sticker#sticker-object
+ */
+export interface APISticker {
+	/**
+	 * ID of the sticker
+	 */
+	id: Snowflake;
+	/**
+	 * For standard stickers, ID of the pack the sticker is from
+	 */
+	pack_id?: Snowflake;
+	/**
+	 * Name of the sticker
+	 */
+	name: string;
+	/**
+	 * Description of the sticker
+	 */
+	description: string;
+	/**
+	 * For guild stickers, the name of a unicode emoji representing the sticker's expression. for standard stickers, a comma-separated list of related expressions.
+	 */
+	tags: string;
+	/**
+	 * Previously the sticker asset hash, now an empty string
+	 * @deprecated
+	 */
+	asset: '';
+	/**
+	 * Type of sticker
+	 *
+	 * See https://discord.com/developers/docs/resources/sticker#sticker-object-sticker-types
+	 */
+	type: StickerType;
+	/**
+	 * Type of sticker format
+	 *
+	 * See https://discord.com/developers/docs/resources/sticker#sticker-object-sticker-format-types
+	 */
+	format_type: StickerFormatType;
+	/**
+	 * Whether or not the guild sticker is available
+	 */
+	available?: boolean;
+	/**
+	 * ID of the guild that owns this sticker
+	 */
+	guild_id?: Snowflake;
+	/**
+	 * The user that uploaded the guild sticker
+	 */
+	user?: APIUser;
+	/**
+	 * The standard sticker's sort order within its pack
+	 */
+	sort_value?: number;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/sticker#sticker-object-sticker-types
+ */
+export const enum StickerType {
+	/**
+	 * An official sticker in a pack, part of Nitro or in a removed purchasable pack
+	 */
+	STANDARD = 1,
+	/**
+	 * A sticker uploaded to a Boosted guild for the guild's members
+	 */
+	GUILD,
+}
+
+/**
+ * https://discord.com/developers/docs/resources/channel#sticker-object-sticker-format-types
+ */
+export const enum StickerFormatType {
+	PNG = 1,
```

Comment:
Ditto

Reply:
this was already capitalized before, I'll change it